### ISSUE TITLE
tag job is failing for some Java repos

### ISSUE THREAD

**Issue by bcoe** [googleapis/releasetool#346]
This behavior seems similar to what @Neenu1995 described yesterday:

```
Working set:
 * chore: release v1.3.0-alpha: https://github.com/googleapis/java-compute/pull/488
 * chore: release 0.118.1-SNAPSHOT: https://github.com/googleapis/java-compute/pull/485
 * chore: release 0.157.0: https://github.com/googleapis/java-cloud-bom/pull/2269
Processing chore: release v1.3.0-alpha: https://github.com/googleapis/java-compute/pull/488
command github-release failed with status 404
---------
GitHubAPIError: Not Found
    at GitHub.getFileContentsWithSimpleAPI (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/github.js:1379:23)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /tmpfs/src/github/releasetool/node_modules/release-please/build/src/github.js:385:24
    at async GitHub.getFileContentsOnBranch (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/github.js:1372:20)
    at async GitHub.getFileContents (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/github.js:1325:16)
    at async JavaYoshi.buildReleaseForVersion (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/release-pr.js:338:36)
    at async GitHubRelease.createRelease (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/github-release.js:35:25)
    at async GitHubRelease.run (/tmpfs/src/github/releasetool/node_modules/release-please/build/src/github-release.js:47:38)
CalledProcessError(1, ['npx', 'release-please', 'github-release', '--token=/tmp/tmp32nfzx7x', '--default-branch=master', '--release-type=java-yoshi', '--bump-minor-pre-major=true', '--repo-url=googleapis/java-compute', '--package-name=', '--debug'])
```

**Comment by Neenu1995** [comment]
The tag jobs did fail yesterday evening for java-compute(v1.3.0), java-cloud-bom(v0.157.0) and one other repo. I manually tagged all these releases. Hence these jobs should be cancelled and not retriggered.
release 0.118.1-SNAPSHOT for java compute is a SNAPSHOT release and does not require tagging.

Also, the failure in tag jobs could not have been caused by any java specific issue and is likely caused by the kokoro periodic job failure.
